build.js

```diff
@@ -20,7 +20,7 @@ const tickSvg = '<svg xmlns="http://www.w3.org/2000/svg" height="48" width="48">
 const partialSvg = '<svg xmlns="http://www.w3.org/2000/svg" height="48" width="48"><path d="M10.4 26.4Q9.4 26.4 8.7 25.7Q8 25 8 24Q8 23 8.7 22.3Q9.4 21.6 10.4 21.6Q11.4 21.6 12.1 22.3Q12.8 23 12.8 24Q12.8 25 12.1 25.7Q11.4 26.4 10.4 26.4ZM24 26.4Q23 26.4 22.3 25.7Q21.6 25 21.6 24Q21.6 23 22.3 22.3Q23 21.6 24 21.6Q25 21.6 25.7 22.3Q26.4 23 26.4 24Q26.4 25 25.7 25.7Q25 26.4 24 26.4ZM37.6 26.4Q36.6 26.4 35.9 25.7Q35.2 25 35.2 24Q35.2 23 35.9 22.3Q36.6 21.6 37.6 21.6Q38.6 21.6 39.3 22.3Q40 23 40 24Q40 25 39.3 25.7Q38.6 26.4 37.6 26.4Z"/></svg>';
 
 const sdkManifestSuffixes = [
-  'java',
+  'java', 'ruby',
```

Comment:
```suggestion
  'java',
  'ruby',
```
Very minor suggestion; slightly more readable this way and will be better for git merges/blames etc